### PR TITLE
Parse a Resolved Dependencies json File for Go

### PR DESCRIPTION
**Pull request by mikeyoung85**

### CODE REVIEW

File: lib/bibliothecary/parsers/go.rb [hunk 1]
```diff
@@ -140,6 +144,10 @@ def self.parse_go_sum(file_contents)
         deps.uniq
       end
 
+      def self.parse_go_resolved(file_contents)
+        JSON.parse(file_contents).map { |dep| { name: dep["Path"], requirement: dep["Version"], type: 'runtime' } }
```
Comment (katzj) [inline lib/bibliothecary/parsers/go.rb:148]: I think we want to skip the ones that are `Main`

Comment (mikeyoung85) [reply lib/bibliothecary/parsers/go.rb:148]: I'll add a filter in here

File: lib/bibliothecary/parsers/go.rb [hunk 2]
```diff
@@ -140,6 +144,12 @@ def self.parse_go_sum(file_contents)
         deps.uniq
       end
 
+      def self.parse_go_resolved(file_contents)
+        JSON.parse(file_contents)
+          .select { |dep| dep["Main"] == "false" }
```
Comment (jsonperl) [inline lib/bibliothecary/parsers/go.rb:149]: Shouldn't matter but maybe `dep["Main"] != "true"` ?

**Review by jsonperl — APPROVED** [review]
Wonderful! 🎉

**Review by jsonperl — CHANGES_REQUESTED** [review]
Oh actually, don't forget a Version bump.

**Pull request merged**